twig/BS3/fields/toggleTrueFalse.twig

```diff
@@ -1,14 +1,25 @@
 {% macro render(field) %}
-    <div>
-        <div class="btn-group" id="{{ field.id }}" data-toggle="buttons">
+    <fieldset>
+        {% if field.label is not null %}
```

Comment:
These inline overrides like this cause duplication.  We most likely don't want every heading twice.